src/reducers/MapReducer.js

```diff
@@ -35,6 +35,26 @@ const INITIAL_STATE = {
     selectedSite: {}
 };
 
+const getBoundingBoxForRegion = ({region}) => {
+    return {
+        west: region.longitude - (region.longitudeDelta / 2), // westLng - min lng
+        south: region.latitude - (region.latitudeDelta / 2), // southLat - min lat
+        east: region.longitude + (region.longitudeDelta / 2), // eastLng - max lng
+        north: region.latitude + (region.latitudeDelta / 2) // northLat - max lat
+    }
+};
```

Comment:
Turns out `react-native-maps` doesn't have a native way to check that things are in the `boundingBox` (visible map area), so I found this heuristic that someone was using and it seems to work pretty well.

Reply:
cool,  how does that work?